urllib3/exceptions.py

```diff
@@ -144,6 +144,11 @@ class ResponseError(HTTPError):
     SPECIFIC_ERROR = 'too many {status_code} error responses'
 
 
+class SSLConfigurationWarning(HTTPWarning):
```

Comment:
I don't think TLS compression is disabled on Pythons without ``SSLContext`` nor is it possible to disable SSLv3 without ``SSLContext``. Python 2.7 (at least some versions) have SSLv2 disabled, I'm not sure if 2.6 does or if all versions of 2.7 does.

I think it's pretty fair to say that without ``SSLContext`` the security of the connection is likely to be less secure unless the platform has disabled these items inside of OpenSSL itself.